src/backend/joanie/tests/payment/test_backend_lyra.py

```diff
@@ -168,6 +168,45 @@ def test_payment_backend_lyra_get_one_click_payment_data(self):
             },
         )
 
+    def test_payment_backend_lyra_get_payload_data_capture(self):
+        """
+        Lyra backend has `_get_payload_data` method which should
+        return the common payload to create a payment or a one click payment.
```

Comment:
Are we still in the case where we want to create a payment or a one click payment ?